testing/idris/APKBUILD

```diff
@@ -0,0 +1,44 @@
+#-*-mode: Shell-script; coding: utf-8;-*-  
+# Maintainer: Mitch Tishmack <mitch.tishmack@gmail.com>
+pkgname=idris
+pkgver=1.0
+pkgrel=0
+pkgdesc="Idris - A Language with Dependent Types"
+url="http://www.idris-lang.org"
+arch="x86_64 armhf"
+license="bsd3"
+depends="libffi gmp ncurses libffi-dev gmp-dev"
+makedepends="bash ghc ghc-dev cabal libffi gmp libffi-dev gmp-dev ncurses-dev zlib-dev"
```

Comment:
Subpackage -dev should always depend on the origin package (e.g. libffi-dev on libffi), so it’s not needed to add both libffi and libffi-dev to makedepends, only libffi-dev. The same applies for gmp, but currently not for ghc (ghc pkg should be fixed).